crates/swc_ecma_lints/src/rules/mod.rs

```diff
@@ -6,10 +6,25 @@ use swc_ecma_visit::{noop_fold_type, Fold};
 mod const_assign;
 mod duplicate_bindings;
 mod duplicate_exports;
+pub mod no_alert;
 pub mod no_console;
```

Comment:
Also, I missed this one on previous review. Can you remove `pub` here, too?

Reply:
changed in on pub(crate). Can't remove `pub` because config structs placed into modules